### PR TITLE
LP-2781 Add structure for Admin Dashboard

### PR DESCRIPTION
**Pull request by asadali145**
[LP-2781](https://philanthropyu.atlassian.net/browse/LP-2781)

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2333/workflows/workflows_id/jobs/2333/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/admin_dashboard/admin_settings/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/lms/admin_dashboard/admin_settings/urls.py (100%)
openedx/adg/lms/admin_dashboard/applications/urls.py (100%)
openedx/adg/lms/admin_dashboard/performance/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/lms/admin_dashboard/performance/urls.py (100%)
openedx/adg/lms/admin_dashboard/urls.py (100%)
openedx/adg/lms/admin_dashboard/webinars/urls.py (100%)
openedx/adg/lms/applications/models.py (50.0%): Missing lines 109,422
openedx/adg/lms/applications/tests/test_models.py (100%)
openedx/adg/lms/webinars/models.py (0.0%): Missing lines 330
-------------
Total:   27 lines
Missing: 9 lines
Coverage: 66%
-------------
```

**Comment by asadali145** [comment]
Please ignore failing unit test. It failing due to the low coverage.

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2337/workflows/workflows_id/jobs/2337/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/admin_dashboard/admin_settings/urls.py (100%)
openedx/adg/admin_dashboard/applications/urls.py (100%)
openedx/adg/admin_dashboard/performance/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/admin_dashboard/performance/urls.py (100%)
openedx/adg/admin_dashboard/urls.py (100%)
openedx/adg/admin_dashboard/webinars/urls.py (100%)
-------------
Total:   21 lines
Missing: 6 lines
Coverage: 71%
-------------
```

File: openedx/adg/admin_dashboard/performance/urls.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from django.urls import path, re_path
+from django.views.generic import TemplateView
+
+urlpatterns = [
+    path(
+        '',
+        TemplateView.as_view(template_name='adg/admin_dashboard/performance/performance_tracking.html')
+    ),
+    re_path(
+        r'{}/grades/'.format(settings.COURSE_KEY_PATTERN),
```
Comment (AliAdnanSohail) [inline openedx/adg/admin_dashboard/performance/urls.py:14]: can we do it something like this?

`rf'{settings.COURSE_KEY_PATTERN}/grades/'`

Comment (asadali145) [reply openedx/adg/admin_dashboard/performance/urls.py]: Never tried it, But yes we can. Updated!

**Review by AliAdnanSohail — APPROVED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2338/workflows/workflows_id/jobs/2338/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/admin_dashboard/admin_settings/urls.py (100%)
openedx/adg/admin_dashboard/applications/urls.py (100%)
openedx/adg/admin_dashboard/performance/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/admin_dashboard/performance/urls.py (100%)
openedx/adg/admin_dashboard/urls.py (100%)
openedx/adg/admin_dashboard/webinars/urls.py (100%)
-------------
Total:   21 lines
Missing: 6 lines
Coverage: 71%
-------------
```

**Review by harris-irfan — CHANGES_REQUESTED** [review]

File: openedx/adg/admin_dashboard/admin_settings/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+"""
```
[the remark this answers is not in the document]
Comment (asadali145) [reply openedx/adg/admin_dashboard/admin_settings/__init__.py:1]: Did this at first but `'settings' conflicts with the name of an existing Python module and cannot be used as an app name. Please try another name.`

**Review by harris-irfan — APPROVED** [review]
Please remove the following migrations directories and files as they are auto-generated upon initial migration

- openedx/adg/admin_dashboard/admin_settings/migrations/__init__.py
- openedx/adg/admin_dashboard/performance/migrations/__init__.py
- openedx/adg/admin_dashboard/webinars/migrations/__init__.py

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2341/workflows/workflows_id/jobs/2341/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/admin_dashboard/admin_settings/urls.py (100%)
openedx/adg/admin_dashboard/applications/urls.py (100%)
openedx/adg/admin_dashboard/performance/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/admin_dashboard/performance/urls.py (100%)
openedx/adg/admin_dashboard/urls.py (100%)
openedx/adg/admin_dashboard/webinars/urls.py (100%)
-------------
Total:   21 lines
Missing: 6 lines
Coverage: 71%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2343/workflows/workflows_id/jobs/2343/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/admin_settings/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/admin_dashboard/admin_settings/urls.py (100%)
openedx/adg/admin_dashboard/applications/urls.py (100%)
openedx/adg/admin_dashboard/performance/apps.py (0.0%): Missing lines 4,7-8
openedx/adg/admin_dashboard/performance/urls.py (100%)
openedx/adg/admin_dashboard/urls.py (100%)
openedx/adg/admin_dashboard/webinars/urls.py (100%)
-------------
Total:   21 lines
Missing: 6 lines
Coverage: 71%
-------------
```

**Pull request merged**